src/pages/version/taskDuration/TaskDurationTable.tsx

```diff
@@ -1,24 +1,23 @@
-import styled from "@emotion/styled";
-import { palette } from "@leafygreen-ui/palette";
-import { Table, TableHeader } from "@leafygreen-ui/table";
+import { useCallback, useMemo, useRef } from "react";
+import { useLeafyGreenTable } from "@leafygreen-ui/table/new";
```

Comment:
I think we can get rid of the non-`/new` dependency in this PR!

Reply:
soo true...